tests/e2e/helper.go

```diff
@@ -30,13 +30,14 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 
 	config = pubnub.NewConfig()
-	config.PublishKey = "pub-c-afeb2ec5-45e9-449f-9a8d-c4940a9c7836"
-	config.SubscribeKey = "sub-c-e41d50d4-43ce-11e8-a433-9e6b275e7b64"
+	config.PublishKey = "pub-c-c1648ded-d156-4a2d-9dbb-a23262945fe2"
```

Comment:
I wish this didn't have to be hardcoded. :(

Reply:
required for integration tests, the keys will be disabled after deploy.

Reply:
Will make changes after the current tasks.